index.html

```diff
@@ -15,31 +15,294 @@
+                    </div>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+
+        </section>
+        <section class="About-me">
+            <div class="aboutme-text">
+                <div class="about-title">
+                    <h1>About me</h1>
+                </div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, </span>
+                    <br /> don’t hestiate to contact me.
+                </p>
+                <button class="aboutme-button">Get My Resume</button>
+            </div>
+            <div class="aboutme-card">
+                <div class="aboutme-card-works-inactive">
+                    <div class="aboutme-img-plaholder">
+                        <img src="/Images/Img Plaholder1.png" alt="ImageBacc" />
+                    </div>
+                    <div class="aboutme-icons">
+                        <div class="aboutme-iconlanguages">
+                            <img alt="Ellipse151411" src="/Images/frameworks.png" class="aboutme-ellipse15" />
+                            <div class="aboutme-icon-langguage">
+                                <div class="aboutme-frame">
+                                    <div class="aboutme-group">
+                                        <img src="/Images/Group12.png" alt="circle">
+
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="aboutme-titleand-tag">
```

Comment:
- [ ] I kindly suggest that you also use a two level nested `ul` `li’`s for your Language, Framework, Skills. 

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```

Please note that the above is just an example and may not apply directly to your code.